src/appleseed/renderer/modeling/bsdf/lambertianbrdf.cpp

```diff
@@ -175,6 +177,7 @@ namespace
     };
 
     typedef BSDFWrapper<LambertianBRDFImpl> LambertianBRDF;
+    typedef MicrofacetBRDFWrapper<LambertianBRDFImpl> MicrofacetGlossyBRDF;
```

Comment:
This should be `MicrofacetLambertianBRDF`

Reply:
Fixed. Also, fixed the factory below.